ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,34 @@
+(Press Preview to see how the issue will be after creating it and to easier read the template.)
+
+# Preconditions
+Have you read our [brilliant documentation](https://signering-docs.readthedocs.io/) before posting the issue? We are continuously improving it, so be sure to give it a try before asking for help. 
+
+# Issue title
+
+Please give a descriptive title for the issue in the field above. 
+
+A good example is: `Unable to create letter with one attachement `.
+A bad example is: `Please help. Important for integration`
+
+> Delete this chapter after you have created your issue title.
+
+# Description
+
+Please give a good explanation of the issue you are experiencing. An issue description **must** contain the following:
+1. A summary and a more detailed description of the problem at hand.
+2. The exact code that gives the error. Remember to remove passwords to enterprise certificate, names etc. Use ``` before and after the code to create examples like this:
+
+``` java
+public void myCode(){
+  doCall();
+}
+```
+
+3. Please provide the error message you get when performing the action.
+4. Provide us with what you have tried so far.
+
+## Do not:
+- Use screenshots or PDF's to describe the issue.
+- Assume we know everything about your code. We need context to understand your problem.
```

Comment:
Foreslår å skrive om her: Sløyf overskrift, begynn hver setning med "Do not" for å gjøre det mer kraftfullt (og lettere å lese gjennom raskt)